frontend/src/slices/projects/projects.slice.ts

```diff
@@ -7,17 +7,20 @@ import { type ProjectGetAllItemResponseDto } from '~/packages/projects/projects.
 import {
   createProject,
   getUserProjects,
+  setCurrentProject,
   uploadProjectImage,
 } from './actions.js';
 
 type State = {
   dataStatus: ValueOf<typeof DataStatus>;
   projects: ProjectGetAllItemResponseDto[];
+  currentProject: ProjectGetAllItemResponseDto | null;
 };
 
 const initialState: State = {
   dataStatus: DataStatus.IDLE,
   projects: [],
+  currentProject: null,
```

Comment:
you need to load at all the time when user opens the site page
LOAD. not just take from reducer